### PR TITLE
docker/podman solution message for non-amd64 archs

### PR DESCRIPTION
**Pull request by medyagh**
adds solution message for https://github.com/kubernetes/minikube/issues/7818

### CODE REVIEW

File: pkg/minikube/registry/drvs/docker/docker.go [hunk 1]
```diff
@@ -71,6 +71,10 @@ func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
 
 func status() registry.State {
 	docURL := "https://minikube.sigs.k8s.io/docs/drivers/docker/"
+	if runtime.GOARCH != "amd64" {
+		return registry.State{Error: fmt.Errorf("doker driver is not supported on %q systems yet", runtime.GOARCH), Installed: false, Healthy: false, Fix: "Try other drivers", Doc: docURL}
```
Comment (afbjorklund) [inline pkg/minikube/registry/drvs/docker/docker.go:75]: typo: docker

File: site/content/en/docs/drivers/podman.md [hunk 2]
```diff
@@ -11,7 +11,14 @@ aliases:
...
 {{% /pageinfo %}}
 
-The podman driver is another kubernetes in container driver for minikube. similar to [docker](https://minikube.sigs.k8s.io/Drivers/docker/) driver. The podman driver is  experimental, and only supported on Linux and macOS (with a remote podman server)
+The podman driver is another kubernetes in container driver for minikube. similar to [docker](https://minikube.sigs.k8s.io/Drivers/docker/) driver. The podman driver is  experimental, and only supported on Linux and macOS (with a remote podman server).
+
+
+
+## Requirements
+- amd64 system 
+- Latest version of [Podman](https://podman.io/getting-started/installation)
```
Comment (afbjorklund) [inline site/content/en/docs/drivers/podman.md:20]: Shouldn't be required with latest version, then we will shut out platforms such as RHEL.

Probably should just say something like "Install Podman", similar to what Docker says ?

**Comment by medyagh** [comment]
thanks @afbjorklund addressed the review comments.

**Comment by afbjorklund** [comment]
Looks good to me! I think it will be easier to do kicbase for other arch, then to do the minikube.iso

**Pull request merged**